src/Models/MakeModel.php

```diff
@@ -40,6 +40,24 @@ public function checkDistinct($column, $value) : bool
         return ($res['count'] === 0);
       
     }
+
+    public function getAllMakes() : array
+    {
+        $query = $this->db->prepare("SELECT * FROM `make`");
+        $query->execute();
+        $res = $query->fetchAll();
+        if (!$res)
+        {
+            return [];
+        }
```

Comment:
fetchAll will only ever return an array, never any other type (https://www.php.net/manual/en/pdostatement.fetchall.php) 

So doing this is not needed, you can just take it out entirely. If no results are found, it will naturally be an empty array